### DISCUSSION TITLE
Is on the plans to create a Dreambooth training for Lora?

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#2563] [Q&A]
We're waiting a few months for this, just would like to know if is on the plans to create the Dreambooth training for Lora. The one I know is so complicated that I always mess.

**Comment by TheLastBen** [comment]
I made SDXL LoRA trainer notebook in Paperspace and Runpod

**Reply by meunumerotim333** [reply]
Nice didn't know. THank you! :) 

Would be possible to update on the future do Stable 1.5 and Free Colab?

**Reply by TheLastBen** [reply]
possibly but I can't promise

**Reply by cutecoelho** [reply]
I would love it too. Most of users still using 1.5 models. Wish have training Lora more accessible, all notebook existed is complicated.

**Reply by TheLastBen** [reply]
soon
